components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -1980,6 +1980,42 @@ public void deleteIdP(String idPName, int tenantId, String tenantDomain)
         }
     }
 
+    public void forceDeleteIdP(String idPName,
+                               int tenantId,
```

Comment:
Shall we reformat here ?

Reply:
It was a choice between
public void deleteIdP(String idPName, int tenantId, String tenantDomain)
            throws IdentityProviderManagementException {